OSDOCS#9316:Correct GCE in-tree migration info (#70251)
openshift/openshift-docs: storage/persistent_storage/persistent-storage-gce.adoc

[IMPORTANT]
====

> {product-title} 4.12 and later provides automatic migration for GCE Persist Disk in-tree volume plugin to its equivalent CSI driver.

Reviewer: ```suggestion
{product-title} 4.12 and later provides automatic migration for the GCE Persist Disk in-tree volume plugin to its equivalent CSI driver.
```

CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes.
ifndef::openshift-dedicated[]
For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
endif::openshift-dedicated[]

> In-tree plugins will eventually be removed in future versions of {product-title}.

Reviewer: ```suggestion
In-tree plugins are expected to be removed in future versions of {product-title}.
```
See https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future
